policy/policy.go

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2016 Pani Networks
+// All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package policy
+
+import (
+	"errors"
+	"fmt"
+	"github.com/romana/core/common"
+	"github.com/romana/core/tenant"
+	"log"
+	//	"net"
+)
+
+// Policy provides Policy service.
```

Comment:
Aside from the fact that the docstring isn't particularly helpful ( :-) ), it seems to be incorrect according to the Go style guides: The docstring should start with the name of the entity, so in this case it should be "PolicySvc..."